### PR TITLE
Fix release script to have everything we need

### PR DESCRIPTION
**Pull request by MaitreyaBuddha**
Release script needed to add changes and push correctly.

### CODE REVIEW

File: Tools/create-release.bash [hunk 1]
```diff
@@ -11,7 +11,12 @@ set -o errexit
 TYPE=$1
 
 release_version=$(./Tools/create-release.py ${TYPE})
+cat AndroidSDKCore/sdk-version.txt
```
Comment (e7mac) [inline Tools/create-release.bash]: what would this line do?

Comment (e7mac) [reply Tools/create-release.bash]: Are we adding the version in this file somehow?

File: Tools/create-release.bash [hunk 2]
```diff
@@ -11,7 +11,12 @@ set -o errexit
...
 
-# create a branch, push on success
-git checkout -b "release/${release_version}"
-git push --set-upstream origin "release/${release_version}"
+git reset HEAD --
+RELEASE_BRANCH=release/${release_version}
+git checkout -b ${RELEASE_BRANCH}
+
+git add AndroidSDKCore/sdk-version.txt
+git commit -m "Created ${TYPE} release ${release_version}"
```
Comment (Cyril-Engels) [inline Tools/create-release.bash]: `git commit -m "Create ${TYPE} release ${release_version}."` Present tense?

File: Tools/create-release.bash (hunk 1 above)
Comment (MaitreyaBuddha) [reply Tools/create-release.bash]: Yes that file is already in to note our current version. It's read during build time to be hard-coded in the binary. This line `cat...` Just displays the version. It's a debug line but I think useful.

**Pull request merged**